docs/DeveloperGuide.md

```diff
@@ -36,3 +44,64 @@
 ## Instructions for manual testing
 
 {Give instructions on how to do a manual product testing e.g., how to load sample data to be used for testing}
+
+## Parser Component
+
+Here is a general guide of how Parse Class looks like in sequence diagram.
+
+![Parse.png](diagram%2FParse.png)
+
+Here is a class diagram for the method changeTaskTiming method in Parser Class.
+
+![changeTaskTimingClassDiagram.png](diagram%2FchangeTaskTimingClassDiagram.png)
```

Comment:
Should the (C) logo be in the class diagrams? Do include `hide circle` in the `puml` file to disable it :)